docs/config.rst

```diff
@@ -0,0 +1,122 @@
+Configuration Options
+=====================
+``schema.registry.url``
+ URL for schema registry
+
+ * Type: string
+ * Default:
+ * Importance: high
+
+``max.schemas.per.subject``
+ The max number of schema objects per subject
```

Comment:
I had to go digging in the schema-registry code to understand what this meant. It seems like it's mostly a sanity check to stop registration if you generate too many schemas for a topic, is that accurate? If so, probably low importance since we expect the default to work well. The description could be longer and more helpful, explaining what it accomplishes, not just what it limits.